doc/source/dev/contributor/quickstart_gitpod.rst

```diff
@@ -0,0 +1,136 @@
+:orphan:
+
+.. _quickstart-gitpod:
+
+=======================================================
+Development environment quickstart guide (Gitpod)
+=======================================================
+
+This quick start guide covers:
+
+*  using GitPod for your SciPy development environment
+*  creating a personal fork of the SciPy repository on GitHub
+*  a quick tour of Gitpod and VSCode
+*  working on the SciPy documentation in SciPy
+
+Gitpod
+-------
+
+`Gitpod`_  is an open-source platform for automated and ready-to-code development environments. It enables developers to describe their dev environment as code and start instant and fresh development environments for each new task directly from your browser. This reduces the need to install local development environments and deal with incompatible dependencies.
+
+To be able to use Gitpod, you will need to log in with your GitHub account, so if 
+you do not have an account yet, you will need to create one first. 
+
+Head over to the `Gitpod`_ website and click on the **Continue with GitHub** button. You will be redirected to GitHub authentication, which will allow you to use Gitpos with your GitHub repositories.
+
+Forking the SciPy repository
+-----------------------------
+
+The best way to work on SciPy as a contributor is by making a fork of the repository first. 
+
+#. Browse to the `SciPy repository on GitHub`_ and `create your own fork`_.
+
+#. Browse to your fork. Your fork will have a URL like https://github.com/andyfaff/scipy, except with your GitHub username in place of "andyfaff".
+
+Starting Gitpod
+----------------
+Once you have authenticated to Gitpod through GitHub, you will see a **Gitpod** button next to the "Code" button on the repository.
```

Comment:
I'm missing this I think?

![image](https://user-images.githubusercontent.com/7903078/114955152-3f66ba00-9e19-11eb-9ccb-ba9b78a51d12.png)

![image](https://user-images.githubusercontent.com/7903078/114955191-4ee60300-9e19-11eb-9773-459db75c87f5.png)

![image](https://user-images.githubusercontent.com/7903078/114955213-5dccb580-9e19-11eb-8711-8167c14b2c40.png)

![image](https://user-images.githubusercontent.com/7903078/114955224-691fe100-9e19-11eb-8371-1dab62d41993.png)

Reply:
Do I need to adjust/override the install from the GitHub side?

Reply:
Ah gotcha - you need to have public_repo permissions. I this does happen when you first register and install the hooks for GiHub 🤔  


![Gitpod_-_Dashboard](https://user-images.githubusercontent.com/23552331/115005274-9b742180-9e9f-11eb-9bdb-0c1f0b956dea.png)

Reply:
@rgommers can you remember if there was anything else needed?

Reply:
Ok, that makes sense--I think it wanted private repo access as well initially (for the override) so I chickened out on installing right this moment, but I suspect that would fix it.

Reply:
You can always update your settings directly from the gitpod.io integration dashboard (public and private repo)